docs/book/src/quick-start.md

```diff
@@ -24,64 +32,139 @@ sudo mv /tmp/kubebuilder_2.0.1_${os}_${arch} /usr/local/kubebuilder
+logic. For more info see [Designing an API](/cronjob-tutorial/api-design.md) and [What's in
+a Controller](cronjob-tutorial/controller-overview.md).
+
+<details><summary>Click here to see an example. `(api/v1/guestbook_types.go)` </summary>
+<p>
+
+```go 
+// GuestbookSpec defines the desired state of Guestbook
+type GuestbookSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+    // Quantity of instances
+    // +kubebuilder:validation:Minimum=1
+    // +kubebuilder:validation:Maximum=10
+    Size int32 `json:"size"`
+    
+    // Name of the ConfigMap for GuestbookSpec's configuration
+    // +kubebuilder:validation:MaxLength=15
+    // +kubebuilder:validation:MinLength=1
+    ConfigMapName string `json:"configMapName"`
+    
+    // +kubebuilder:validation:Enum=Phone,Address,Name
+    Type string `json:"alias,omitempty"`
+    
+	// TLS policy of Guestbook nodes
+	TLS *TLSPolicy `json:"TLS,omitempty"`
+}
+
+// GuestbookStatus defines the observed state of Guestbook
+type GuestbookStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+	
+	// GuestbookStatus is the set of Guestbook node specific statuses: Active or Standby
+    GuestbookStatus GuestbookStatus `json:"guestbookStatus"`
+    
+    // Status of Guestbook Nodes
+    Nodes []string `json:"nodes"`
+}
+
+type GuestbookStatus struct {
+
+	// PodName of the active Guestbook node. 
+	Active string `json:"active"`
+
+	// PodNames of the standby Guestbook nodes.
+	Standby []string `json:"standby"`
+}	
+```
+
+</p>
+</details>
 
-**Optional:** Edit the API definition or the reconciliation business
-logic. For more on this see [What's in
-a Controller](cronjob-tutorial/controller-overview.md) and [Designing an
-API](/cronjob-tutorial/api-design.md).
 
-## Test It Out Locally
+## Test It Out 
 
 You'll need a Kubernetes cluster to run against.  You can use
 [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or
 run against a remote cluster.
 
+<aside class="note">
+<h1>Context Used</h1>
+
 Your controller will automatically use the current context in your
 kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
 
-Install the CRDs into the cluster:
+</aside> 
 
+Install the CRDs into the cluster:
 ```bash
 make install
 ```
 
 Run your controller (this will run in the foreground, so switch to a new
 terminal if you want to leave it running):
-
 ```bash
 make run
 ```
 
-## Install Samples
+## Install Instances of Custom Resources
 
-Create your samples (make sure to edit them first if you've changed the
+If you pressed `y` for Create Resource [y/n] then you created an (CR)Custom Resource for your (CRD)Custom Resource Definition in your samples (make sure to edit them first if you've changed the
```

Comment:
Since this is giving the user a tour of the system, suggest avoiding "If" statements.

Reply:
But it is optional. IMHO: it is not mandatory for the user to perform the quick start. However, at the end of it was added a suggestion for the user to follow up the tutorial where all these points will be detailed explained. Are you ok with?

Reply:
I would think that 95% or more of use cases for KubeBuilder involve both a CRD and a controller.  
Chosing not to use them is sort of an advanced use case.  So a quickstart (which I see as like a demo), does not need to cover an advanced use case.

Fewer choices for a user in introductory instructions is better.

Reply:
IMHO the goal is to show how to create, build and deploy a project to give a macro idea and not actually cover a basic use case. The basic demo/use case is covered by the tutorial and not here.